files/fr/web/javascript/guide/numbers_and_dates/index.html

```diff
@@ -338,17 +338,21 @@ <h3 id="Méthodes_de_lobjet_Date">Méthodes de l'objet <code>Date</code></h3>
-<span class="comment token">// On définit le jour et le mois</span>
-<span class="keyword token">var</span> finAnnée <span class="operator token">=</span> <span class="keyword token">new</span> <span class="class-name token">Date</span><span class="punctuation token">(</span><span class="number token">1995</span><span class="punctuation token">,</span> <span class="number token">11</span><span class="punctuation token">,</span> <span class="number token">31</span><span class="punctuation token">,</span> <span class="number token">23</span><span class="punctuation token">,</span> <span class="number token">59</span><span class="punctuation token">,</span> <span class="number token">59</span><span class="punctuation token">,</span> <span class="number token">999</span><span class="punctuation token">)</span><span class="punctuation token">;</span>
-<span class="comment token">// On définit l'année avec l'année courante</span>
-finAnnée<span class="punctuation token">.</span><span class="function token">setFullYear</span><span class="punctuation token">(</span>aujourdhui<span class="punctuation token">.</span><span class="function token">getFullYear</span><span class="punctuation token">(</span><span class="punctuation token">)</span><span class="punctuation token">)</span><span class="punctuation token">;</span>
-<span class="comment token">// On calcule le nombre de millisecondes par jour</span>
-<span class="keyword token">var</span> msParJour <span class="operator token">=</span> <span class="number token">24</span> <span class="operator token">*</span> <span class="number token">60</span> <span class="operator token">*</span> <span class="number token">60</span> <span class="operator token">*</span> <span class="number token">1000</span><span class="punctuation token">;</span>
-
-<span class="comment token">// On renvoie le nombre de jours restants dans l'année</span>
-<span class="keyword token">var</span> joursRestants <span class="operator token">=</span> <span class="punctuation token">(</span>finAnnée<span class="punctuation token">.</span><span class="function token">getTime</span><span class="punctuation token">(</span><span class="punctuation token">)</span> <span class="operator token">-</span> aujourdhui<span class="punctuation token">.</span><span class="function token">getTime</span><span class="punctuation token">(</span><span class="punctuation token">)</span><span class="punctuation token">)</span> <span class="operator token">/</span> msParJour<span class="punctuation token">;</span>
-joursRestants <span class="operator token">=</span> Math<span class="punctuation token">.</span><span class="function token">round</span><span class="punctuation token">(</span>joursRestants<span class="punctuation token">)</span><span class="punctuation token">;</span></code></pre>
+<pre class="brush: js line-numbers  language-js"><code class="language-js">var aujourdhui = new Date();
```

Comment:
Il faudrait aussi retirer les attributs `line-numbers`et `language-js` puis l'élément `<code>` (avec la balise fermante qui va bien) car ce sont eux aussi des reliquats du copier-coller.
J'ai conscience que ce n'est pas vraiment dans le descriptif de l'issue :/ mais si on peut faire d'une pierre deux coups :)

Reply:
My bad, I'm still pretty new with MDN tags utility don't hesitate to point out mistakes :smiley:  fixed !